packages/nodes-base/nodes/Twist/Twist.node.ts

```diff
@@ -201,6 +228,193 @@ export class Twist implements INodeType {
 
 					responseData = await twistApiRequest.call(this, 'POST', '/channels/update', body);
 				}
+				//https://developer.twist.com/v3/#archive-channel
+				if (operation === 'archive') {
+					const channelId = this.getNodeParameter('channelId', i) as string;
+					qs.id = channelId;
+
+					responseData = await twistApiRequest.call(this, 'POST', '/channels/archive', {}, qs);
+				}
+				//https://developer.twist.com/v3/#unarchive-channel
+				if (operation === 'unarchive') {
+					const channelId = this.getNodeParameter('channelId', i) as string;
+					qs.id = channelId;
+
+					responseData = await twistApiRequest.call(this, 'POST', '/channels/unarchive', {}, qs);
+				}
+			}
+			if (resource === 'comment') {
+				//https://developer.twist.com/v3/#add-comment
+				if (operation === 'add') {
+					const workspaceId = this.getNodeParameter('workspaceId', i) as string;
+					const threadId = this.getNodeParameter('threadId', i) as string;
+					const content = this.getNodeParameter('content', i) as string;
+					const additionalFields = this.getNodeParameter('additionalFields', i) as IDataObject;
+					const body: IDataObject = {
+						thread_id:threadId,
+						content,
+					};
```

Comment:
You can also fully type `body` here to prevent castings in 260 and 269: `{ threadId: string; content: string; actionsUi?: IDataObject, binaryProperties?: string }`

Reply:
I think fully typed body can be omitted in this situation, as the `actionsUi` & `binaryProperties` are not fixed attributes, they depend on the user input.

Reply:
Properties that may or may not exist (e.g. because they depend on user input) are usually typed with `?` to signal that they are optional. This way, if they exist, the type checker will be able to assist and will not require type castings as we are doing further below. No need to change it - just as reference to keep in mind.